Sources/docc/DocCDocumentation.docc/Reference Syntax/Tutorials Syntax/Top-Level Directives/Tutorials/Resources/Resources.md

```diff
@@ -8,7 +8,7 @@ Displays a set of links to related resources, like downloads, sample code, and v
 
 ## Overview
 
-Use the `Resources` directive to include a Resources section at the bottom of a tutorial's table of contents page. This optional section can offer links to helpful material for continued learning. Provide some text for the top of the section, then add directives,  for example``Documentation``, ``Downloads``, ``Forums``, ``SampleCode``, and ``Videos`) for the resource types you want to share.
+Use the `Resources` directive to include a Resources section at the bottom of a tutorial's table of contents page. This optional section can offer links to helpful material for continued learning. Provide some text for the top of the section, then add directives, for example ``Documentation``, ``Downloads``, ``Forums``, ``SampleCode``, and ``Videos``) for the resource types you want to share.
```

Comment:
I wonder if there's any way we could emit a diagnostic for this? Seems like a missing backtick will be a common typo.

Reply:
That's a good idea but it would probably need to be fixed in Swift Markdown.

The missing backtick makes the link text not parse as its own element. For example, the correct link (below) parses as:

```md
Some text with a ``Link`` followed by some more text.
```

```
Paragraph
├─ Text "Some text with a "
├─ SymbolLink destination: Link
└─ Text " followed by some more text."
```

and a single backtick before and after pages as:

```md
Some text with a `Link` followed by some more text.
```


```
Paragraph
├─ Text "Some text with a "
├─ InlineCode `Link`
└─ Text " followed by some more text."
```

but double backticks before and a single backtick after just parses as a single text element

```md
Some text with a ``Link` followed by some more text.
```
```
Paragraph
└─ Text "Some text with a ``Link` followed by some more text."
```

Reply:
I opened this issue about this https://github.com/apple/swift-markdown/issues/165

Reply:
Great thanks for filing that 👍 ... and I'm guessing there might be more edge cases to handle, like if there is more than one mistake in the same paragraph, or if the author really wanted a single backtick for code voice. Not an obvious fix.